fixture.js

```diff
@@ -16,3 +16,14 @@ var cli = meow({
 Object.keys(cli.flags).forEach(function (el) {
 	console.log(el);
 });
+
+cli.stdinOrInput()
+	.then(function (input) {
+		input.forEach(function (element) {
```

Comment:
The main use-case is one input argument, so would really like to optimize for that, but still support the edge-case of multiple arguments.